src/openassetio-python/tests/package/pluginSystem/resources/entryPoint/README.md

```diff
@@ -0,0 +1,50 @@
+# Entry Point discovery test plugin
```

Comment:
[question] I'm in two minds about whether we want to use "real" packages, which is admittedly more representative, vs. the absolute minimum to test the feature, which is apparently just the plugin module and a `dist-info` directory containing `entry_points.txt`.

It sounds like for privacy purposes we want to delete half the contents anyway.

Reply:
Yeah - I was thinking that too, but wondered if keeping it a "real package" would be more representative when it comes to the fact that parsing is handled by an external lib.

Reply:
Resolving, as discussed in https://github.com/OpenAssetIO/OpenAssetIO/pull/768#discussion_r1036996344